README.rst

```diff
@@ -45,9 +45,9 @@ The ``koji_target`` module can create, update, and delete targets within Koji.
 
     - name: Create a koji build target for Fedora 29
```

Comment:
While we're here we should update this to `Fedora 32` as well.

Reply:
*great* catch!

Reply:
I've force-pushed with this change.